src/timeline/Timeline.js

```diff
@@ -30,7 +30,10 @@ export default class Timeline extends React.PureComponent {
         summary: PropTypes.string.isRequired,
         color: PropTypes.string
       })
-    ).isRequired
+    ).isRequired,
+    scrollToNow: PropTypes.bool,
+    currentDateString: PropTypes.string,
```

Comment:
Let the user pass a Date instead of a string with a specific format so you can validate and format it yourself internally.